docs/user-guide/work-packages/work-package-table-configuration/README.md

```diff
@@ -123,6 +127,28 @@ To switch between the different criteria, open the [work package table configura
 
 ![display-settings](1566397517070.png)
 
+When choosing grouping the work package list by an attribute or by project a button to collapse groups shows up:
+![collapse-button](image-20201211021022685.png)
+
+Use it to quickly collapse or expand all groups at the same time.
+
+## Aggregation by project
+
+You can get a **quick overview of multiple projects** in the Gantt chart. Therefore navigate to the work package module of a project or the [project overarching work package list](../../projects/#project-overarching-reports).
+
+**Group the list** by project by using the work package table configuration (as described above) or by clicking on the small triangle next to "Project" in the table header.
+ ![group-by-project](image-20201211020614221.png)
+
+**Display the [Gantt chart](../../gantt-chart)** by clicking on the button in the upper right corner.
+![insert-gantt-chart-button](image-20201211020748715.png)
+
+Use the minus next to the project's name or the **collapse button** in the upper right corner to collapse some or all projects.
+
+This will give you an **aggregated view of the projects' milestones**.
+
+**Please note**: If you want to make use of this feature, it is necessary to add milestones for the most important dates to your projects. At the moment this feature is not available for other [work package types](../../../getting-started/work-packages-introduction/#what-is-a-work-package). Which projects are shown depends on the [filters](#filter-work-packages) of the work package table you're in and your [permissions](../../../system-admin-guide/users-permissions/roles-permissions/).
```

Comment:
We can change "Which projects are shown depends on the filters of the work package table you're in and your permissions." to something like "The list of displayed projects depends on your permissions. You can only see private projects that you are a member of and public projects."